modules/mono/glue/GodotSharp/GodotSharp/Core/GD.cs

```diff
@@ -334,6 +335,21 @@ public static void PrintT(params object[] what)
             NativeFuncs.godotsharp_printt(godotStr);
         }
 
+        [StackTraceHidden]
+        private static void ErrPrintError(string message, godot_error_handler_type type = godot_error_handler_type.ERR_HANDLER_ERROR)
+        {
+            // Skip 1 frame to avoid current method.
```

Comment:
I still get `GD.PushError` instead of its caller as the location of the error.

Reply:
You mean in the root tree item? The `<C# Source>` item? Or the first item of `<Stack Trace>`? See the image attached to the PR description. The `GD.PushError` calls are still part of the stack trace, are you suggesting to remove them?

Reply:
It looks like this: ![image](https://github.com/godotengine/godot/assets/10944644/f07df8c0-6d3b-4401-80c9-7a93d1c95540) on my end (v4.2.dev.mono.custom_build [13ab2b6f4]) and I would expect it to look like in the PR description. Not sure if you broke something when updating this or if something is different on my system.

Reply:
That's weird, I have checked again and I'm getting the same result as the screenshot in the PR description.

![image](https://github.com/godotengine/godot/assets/3903059/586c719e-838b-4f6b-bc7f-376603bf068d)

Judging from your screenshot, it seems it doesn't consider `Test_Node.cs` to be inside the project, since it's printing the absolute path.

The `ErrPrintError` method should be localizing that path, which makes it a `res://` path, then `ScriptEditorDebugger` checks if the path starts with `res://` to determine if it's inside the project.

Reply:
I started over with a fresh project that doesn't have all kinds of crap in it and the issue still exists.

It's a windows only `localize_path` bug: `GetFileName()` returns a path with `\` as path separators, but while it will convert them to `/` it only does so after checking if the path is local, so you need to either replace `\` before or call localize_path twice.

Reply:
If it's a bug in `localize_path` I'd rather have that fixed in a separate PR, since it seems like you have already fixed it can you open a PR?

Reply:
- Should be resolved by https://github.com/godotengine/godot/pull/79342.